usr/lib/cca_stdll/cca_specific.c

```diff
@@ -299,22 +295,15 @@ static CK_RV cca_key_gen(STDLL_TokData_t *tokdata, TEMPLATE *tmpl,
 static CK_RV init_cca_adapter_lock(STDLL_TokData_t *tokdata)
 {
     struct cca_private_data *cca_private = tokdata->private_data;
-    pthread_rwlockattr_t attr;
+    pthread_rwlockattr_t attr = NULL;
```

Comment:
I guess you change this back in a follow up commit. Please squash that one with this commit to not make the change at all.